src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -29,6 +30,18 @@ class ConsignmentService(
     consignments.map(rows => rows.headOption.map(row => Consignment(row.consignmentid, row.userid, row.seriesid)))
   }
 
+  def getSeriesOfConsignment(consignmentId: UUID): Future[Option[Series]] = {
+    val consignment: Future[Seq[ConsignmentResult]] = consignmentRepository.getSeriesAndBodyOfConsignment(consignmentId)
+    consignment.map(rows => rows.headOption.map(
+      row => Series(row.series.seriesid, row.series.bodyid, row.series.name, row.series.code, row.series.description)))
+  }
+
+  def getTbOfConsignment(consignmentId: UUID): Future[Option[TransferringBody]] = {
```

Comment:
Could `Tb` be `TransferringBody`? It's more verbose, but I don't think we use TB as an acronym often enough for these to be obvious.

Reply:
Sure!